Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM node:5.10.0
```

Comment:
Hey @urodoz do you know alpine linux? This one here looks pretty solid: https://github.com/mhart/alpine-node